go.mod

```diff
@@ -5,6 +5,7 @@ go 1.15
 require (
 	github.com/beyondstorage/go-service-cos/v2 v2.1.0
 	github.com/beyondstorage/go-service-fs/v3 v3.2.0
+	github.com/beyondstorage/go-service-ipfs v0.0.0-20210630034409-8cf084ff7941
```

Comment:
Maybe `go-service-ipfs` needs to bump a version as we use release versions in `go-storage-example` ?

Reply:
Yes, we need to make a release before we adding them into docs.

@zu1k How about release a `v0.1.0` about `ipfs`?

Reply:
LGTM

Reply:
How about making a release now?